manim/renderer/cairo_renderer.py

```diff
@@ -84,8 +84,12 @@ def play(self, scene, *args, **kwargs):
 
         scene.compile_animation_data(*args, **kwargs)
 
-        # If skip_animations is already True, we can skip all the caching process.
-        if not config["disable_caching"] and not self.skip_animations:
+        if config["disable_caching"]:
+            hash_current_animation = f"uncached_{self.num_plays:05}"
+        if self.skip_animations:
+            logger.debug(f"Skipping animation {self.num_plays}")
+            hash_current_animation = None
+        else :
```

Comment:
```suggestion
        else:
```
Hopefully this will fix the linting problem.

Reply:
I'll commit that

Reply:
Look at that, good job lol

Reply:
@WampyCakes Thanks so much for that, but please ask before commiting on my branch, because it can cause lot of confusion sometimes 😖 

But thanks again nevertheless, that's fine here as it was a very minor change :D